server/container_create.go

```diff
@@ -121,11 +121,12 @@ func (s *Server) createSandboxContainer(containerID string, containerName string
 	// here set it to be "rootfs".
 	specgen.SetRootPath("rootfs")
 
-	args := containerConfig.GetArgs()
-	if args == nil {
-		args = []string{"/bin/sh"}
+	commands := containerConfig.GetCommand()
+	if commands == nil {
+		commands = []string{"/bin/sh"}
```

Comment:
We should set it to /bin/sh (or to command from image config in the future) only when both `args` and `command` are empty. Otherwise, we could possibly get something like `/bin/sh app_to_run` if client will pass `app_to_run` via `args`